tests/test_room_resource/test_create_room_resource.py

```diff
@@ -39,3 +39,13 @@ def test_room_resource_creation_room_id_error(self):
             resource_mutation_0_room_id,
             "Room not found"
         )
+
+    def test_create_resource_in_other_location(self):
```

Comment:
Make this test function name more descriptive. The name is different from what is being tested